kolibri/plugins/epub_viewer/assets/src/views/AddEditCustomThemeModal.vue

```diff
@@ -223,7 +242,6 @@
         } else if (this.showColorPicker == 'linkColor') {
           this.tempTheme.linkColor = color.hex;
         }
```

Comment:
I think you might get an error, or at least a warning, here, if you don't have a return value that's not in an 'if' statement. You may need to add a default `else`

Reply:
done